ovn-tester/ovn_workload.py

```diff
@@ -154,21 +156,37 @@ def configure_localnet(self, physical_net):
         self.run(cmd=f'ovs-vsctl -- set open_vswitch . '
                  f'external-ids:ovn-bridge-mappings={physical_net}:br-ex')
 
+    def configure_external_host_net_str(self, ip, gw):
+        cmd = ''
+        if ip.ip4:
+            cmd += f'ip netns exec ext-ns ip addr add {ip.ip4}/{ip.plen4}' \
+                   f' dev veth0; '
+        if gw.ip4:
+            cmd += f'ip netns exec ext-ns ip route add default via {gw.ip4}; '
+
+        if ip.ip6:
+            cmd += f'ip netns exec ext-ns ip addr add {ip.ip6}/{ip.plen6}' \
+                   f' dev veth0; '
+        if gw.ip6:
+            cmd += f'ip netns exec ext-ns ip route add default via {gw.ip6}; '
+        return cmd
+
     def configure_external_host(self):
```

Comment:
We do all the `DualStackIP` specific expansions in ovn_utils.py for all the other Ctl classes, let's move this one there too.  We could define a new PhysCtl class in ovn_utils.py to configure things on the baremetal/physical host, e.g.:
```
class PhysCtl:
    def __init__(self, sb):
        self.sb = sb

    def run(self, cmd="", stdout=None, timeout=DEFAULT_CTL_TIMEOUT):
        self.sb.run(cmd=cmd, stdout=stdout, timeout=timeout)

    def external_host_provision(self, ip, gw, netns='ext-ns'):
        log.info(f'Adding external host on {self.sb.container}')
        cmd = f'ip link add veth0 type veth peer name veth1; ' \
              f'ip netns add {netns}; ' \
              f'ip link set netns {netns} dev veth0; ' \
              f'ip netns exec {netns} ip link set dev veth0 up; '

        if ip.ip4:
            cmd += f'ip netns exec ext-ns ip addr add {ip.ip4}/{ip.plen4}' \
                   f' dev veth0; '
        if gw.ip4:
            cmd += f'ip netns exec ext-ns ip route add default via {gw.ip4}; '

        if ip.ip6:
            cmd += f'ip netns exec ext-ns ip addr add {ip.ip6}/{ip.plen6}' \
                   f' dev veth0; '
        if gw.ip6:
            cmd += f'ip netns exec ext-ns ip route add default via {gw.ip6}; '

        cmd += 'ip link set dev veth1 up; ' \
               'ovs-vsctl add-port br-ex veth1'

        # Run as a single invocation:
        cmd = f'bash -c \'{cmd}\''
        self.run(cmd=cmd)
```

And remove `configure_external_host() and configure_external_host_net_str() and change `WorkerNode.configure()` to:
```
    def configure(self, physical_net):
        self.configure_localnet(physical_net)
        phys_ctl = ovn_utils.PhysCtl(self)
        phys_ctl.external_host_provision(ip=self.ext_net.reverse(2),
                                         gw=self.ext_net.reverse())
```

Reply:
@dceara ack, I will fix it